gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineTestingIntegrationTest.groovy

```diff
@@ -16,26 +16,27 @@
 
 package com.palantir.baseline
 
-import org.gradle.testkit.runner.BuildResult
-import org.gradle.testkit.runner.TaskOutcome
+import nebula.test.IntegrationSpec
+import nebula.test.functional.ExecutionResult
 import spock.lang.Unroll
 
 @Unroll
-class BaselineTestingIntegrationTest extends AbstractPluginTest {
```

Comment:
Fun fact, this homemade `AbstractPluginTest` framework from 2017 wasted nearly 2 hours of my time because it doesn't run changed code in the test from IntelliJ unless you first recompile the code through gradle 😱 😱 😱 😱 😱 😱 😱